src/main/java/com/minecolonies/coremod/entity/ai/citizen/baker/BakerRecipes.java

```diff
@@ -74,7 +74,7 @@
         inputBread.add(new ItemStack(Items.WHEAT, REQUIRED_WHEAT));
 
         recipes = new ImmutableList.Builder<RecipeStorage>()
-                .add(new RecipeStorage(inputCookie, GRID_SIZE, new ItemStack(Items.COOKIE, 1)))
+                .add(new RecipeStorage(inputCookie, GRID_SIZE, new ItemStack(Items.COOKIE, 6)))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 6 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)